samples/highcharts/plotoptions/treemap-grouping-advanced/demo.js

```diff
@@ -0,0 +1,117 @@
+(async () => {
+
+    const dataJson = await fetch(
```

Comment:
Just `data`, it is no longer JSON after the JSON string has been parsed.
```suggestion
    const dataJson = await fetch(
```